transitfeedflex/README

```diff
@@ -0,0 +1 @@
+Adds Flexible Pick Off / Drop Off (Part of the GTFSFlex specification)
```

Comment:
:+1: That's really cool to add this feature!

Can you please add a link to the documentation of the format ?

Reply:
long time no feedback, sorry :-)
I think it is described here: https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md